extensions/reviewed/Health.json

```diff
@@ -19,96 +19,3851 @@
+              "longDescription": "",
+              "name": "Object",
+              "optional": false,
+              "supplementaryInformation": "",
+              "type": "object"
+            },
+            {
+              "codeOnly": false,
+              "defaultValue": "",
+              "description": "Behavior",
+              "longDescription": "",
+              "name": "Behavior",
+              "optional": false,
+              "supplementaryInformation": "Health::Health",
+              "type": "behavior"
+            }
+          ],
+          "objectGroups": []
+        },
+        {
+          "description": "Change the maximum shield points of the object.",
+          "fullName": "Maximum shield points",
+          "functionType": "Action",
+          "group": "Shield configuration",
+          "name": "SetMaxShield",
+          "private": false,
+          "sentence": "Change the maximum shield of _PARAM0_ to _PARAM2_ points",
+          "events": [
+            {
+              "type": "BuiltinCommonInstructions::Standard",
+              "conditions": [],
+              "actions": [
+                {
+                  "type": {
+                    "value": "Health::Health::SetPropertyMaxShieldPoints"
+                  },
+                  "parameters": [
+                    "Object",
+                    "Behavior",
+                    "=",
+                    "GetArgumentAsNumber(\"Value\")"
+                  ]
+                }
+              ]
+            }
+          ],
+          "parameters": [
+            {
+              "codeOnly": false,
+              "defaultValue": "",
+              "description": "Object",
+              "longDescription": "",
+              "name": "Object",
+              "optional": false,
+              "supplementaryInformation": "",
+              "type": "object"
+            },
+            {
+              "codeOnly": false,
+              "defaultValue": "",
+              "description": "Behavior",
+              "longDescription": "",
+              "name": "Behavior",
+              "optional": false,
+              "supplementaryInformation": "Health::Health",
+              "type": "behavior"
+            },
+            {
+              "codeOnly": false,
+              "defaultValue": "",
+              "description": "Maximum shield",
+              "longDescription": "",
+              "name": "Value",
+              "optional": false,
+              "supplementaryInformation": "",
+              "type": "expression"
+            }
+          ],
+          "objectGroups": []
+        },
+        {
+          "description": "Change shield points. This is not considered damage.",
+          "fullName": "Shield points",
+          "functionType": "Action",
+          "group": "Shield configuration",
+          "name": "SetShieldPoints",
+          "private": false,
+          "sentence": "Change the shield on _PARAM0_ to _PARAM2_ points",
```

Comment:
> This is not considered damage.

The sentence doesn't seem clear.

Reply:
Thanks, I have updated it to:

`Change current shield points. Will not trigger damage cooldown.`
`Change current shield points on _PARAM0_ to _PARAM2_ points`